packages/mui-styled-engine/src/StyledEngineProvider/StyledEngineProvider.test.js

```diff
@@ -49,4 +50,29 @@ describe('[Emotion] StyledEngineProvider', () => {
     );
     expect(rule).to.equal('@layer theme,base,mui,components,utilities;');
   });
+
+  it('should create the same cache once', () => {
```

Comment:
Does this sound more accurate?

```suggestion
  it('should reuse the same cache', () => {
```

Reply:
much better.